app-frontend/src/app/components/colorComposites/colorSchemeDropdown/colorSchemeDropdown.html

```diff
@@ -1,25 +1,28 @@
 <div class="dropdown btn-group"
      uib-dropdown
-     auto-close="outsideClick">
+     auto-close="outsideClick"
+     on-toggle="$ctrl.onDropdownToggle(open)"
+>
   <button class="btn dropdown-label gradient-dropdown"
           uib-dropdown-toggle>
       <div class="gradient-bar gradient-bkg"
-          ng-style="$ctrl.colorSchemeService.colorsToBackground($ctrl.state.scheme.colors)"
+          ng-style="$ctrl.colorSchemeService.colorsToBackground(
+                      $ctrl.reflectedState.scheme.colors, 90, $ctrl.reflectedState.blending.bins > 0)"
```

Comment:
Should this be `$ctrl.reflectedState.blending.bins` (without the 0)? It looks like `colorsToBackground` expects its third parameter to be a number.